### PR TITLE
Neutral classifier background

### PR DESCRIPTION
**Pull request by srallen**
Staging branch URL: https://neutral-classifier-background.pfe-preview.zooniverse.org/

This is follow up to the new project nav. This removes the project background as the default background from the whole project to the background color in the new classifier redesign. This obviously, has a cascading effect, so I've done by best to update any styles where necessary. This includes a start of bringing in more of the styles from the redesign such as removing a lot of button border radiuses, changing the flex max-width on the task area, and some of the colors with the survey task. 

I've already checked with and got an ok from @beckyrother on the changes to the in project context collections, recents, and user profile as well as removing with margin from the project announcement banner.

TO DO:
- [x] Check all the workflow tasks to make sure there aren't any styles that need updating as a consequence of making the task-area background white
- [x] Add the dark background toggle button

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by beckyrother** [comment]
This looks SO GOOD SARAH

I'm writing a message to project owners so they know what's coming before we merge; I'll also have a message to post on Talk. Just want to make sure we've got buy-in from the necessary parties before we push this.

**Comment by srallen** [comment]
@beckyrother I have a basic dark theme setup. The button isn't positioned correctly according to your design because I didn't want to deal with making it responsive, but it works!

https://neutral-classifier-background.pfe-preview.zooniverse.org/projects/mschwamb/planet-four-ridges/classify?env=production

**Comment by srallen** [comment]
The project title font size got updated in #4386 and this hasn't been rebased yet.

**Comment by srallen** [comment]
The main point of this PR is removing the project background image from all the project pages but the home and updating the styles to use several styles from the new classifier design: https://projects.invisionapp.com/d/main#/console/12923997/270495274/preview

**Comment by simoneduca** [comment]
Thanks for clarifying Sarah.

**Comment by srallen** [comment]
Can we hold off on the approval? This is still WIP. I need help checking all the task types to make sure there aren't any broken styles.

File: css/survey-task.styl [hunk 1]
```diff
@@ -36,27 +36,28 @@ $survey-task-pill-button
...
+      background-color:#666666
 
 .survey-task-chooser-characteristic
   display: block
 
 .survey-task-chooser-characteristic-menu
   .modal-form
     background-color: #dddddd
-    border-radius: 6px
+    border-radius: 0px
```
Comment (eatyourgreens) [inline css/survey-task.styl]: Minor point but the syntax is `border-radius: 0` when the value of a property is zero.

File: css/survey-task.styl [hunk 2]
```diff
@@ -246,8 +241,7 @@ $survey-task-pill-button
 
   .modal-form-trigger
     @extends $reset-button
-    background: rgba(232, 232, 232, 1)
-    border-radius: 4px
+    border-radius: 0px
```
Comment (eatyourgreens) [inline css/survey-task.styl]: `border-radius: 0`

**Comment by srallen** [comment]
I think I fixed the few issues I found by clicking through several projects in production. I'll do a check in Firefox, Safari, IE, and Edge next.

**Comment by srallen** [comment]
I keep forgetting we have browserstack setup now. I'll use that.

**Comment by srallen** [comment]
Ok this is ready for a final review. @beckyrother do you have time to coordinate on Monday about announcing these changes and a preview for project owners?

**Comment by beckyrother** [comment]
@srallen yes! I'm going to reach out to some people to add to our testing list, and here's the message I'd like to send to them: https://docs.google.com/document/d/10a3sZQ_zJbUTvfWKMnmAM9saVi3V8vmZp6YdMxwwZr8/edit?usp=sharing

**Comment by beckyrother** [comment]
Just to keep this updated, Cam is working on creating a mailing list and I'm reaching out to volunteers who might be interested in seeing new updates. I'd also like to make a short survey to gather more specific feedback. 

Here's my list so far: https://docs.google.com/spreadsheets/d/1rin1SWvgOwqBP4TUV7Atz9jfEtBusvNf4vHcs_JXYCE/edit?usp=sharing
Any other volunteers we can think of?

**Comment by beckyrother** [comment]
Sorry to be That Person with last-minute design changes, but can the white box around the question have a `solid thin #ebebeb` border? It's subtle but classy ✨ 

i_fancy_cats_ _classify_ _zooniverse

**Comment by srallen** [comment]
@beckyrother I've added the border around the task area.

**Comment by beckyrother** [comment]
Looks great!! Is this ready to send to our beta list?

**Comment by srallen** [comment]
@beckyrother There isn't really a great way to put this behind a feature query parameter like the initial project navbar update since it's touching a lot of shared CSS. I think it's ready to send out. If the users are going to preview it, then we can share the staged link with the production environment query param and ask them to navigate to the projects they work on or own: https://neutral-classifier-background.pfe-preview.zooniverse.org/?env=production

**Comment by beckyrother** [comment]
Cool, let's coordinate on that before we send this to our beta list on Monday. To summarize from #design-thoughts, we'll give testers about a week to submit feedback and then plan to push this live 3/12.

**Comment by srallen** [comment]
Noting here, and as discussed in the design slack channel, I'll be adding a slight adjustment for the navbar height for smaller screen sizes into this PR.

**Comment by beckyrother** [comment]
The design survey did not turn up any issues that won't be fixed with subsequent updates, so this can still go live on Monday (3/12) 👍

**Pull request merged**